agent/reconciler/host_reconciler.go

```diff
@@ -171,6 +207,42 @@ func (r *HostReconciler) getBootstrapScript(ctx context.Context, dataSecretName,
 	return bootstrapSecret, nil
 }
 
+func (r *HostReconciler) getInstallationScript(ctx context.Context, dataSecretName, namespace string) (install, uninstall string, err error) {
+	secret := &corev1.Secret{}
+	err = r.Client.Get(ctx, types.NamespacedName{Name: dataSecretName, Namespace: namespace}, secret)
+	if err != nil {
+		return "", "", err
+	}
+
+	installScript := string(secret.Data["install"])
+	uninstallScript := string(secret.Data["uninstall"])
```

Comment:
Suggestion: This could be moved inline in the `installerController` method

Reply:
We kept it different for `getBootstrapScript` so I implemented in same way. Should I change it?